index.js

```diff
@@ -0,0 +1,167 @@
+const express = require('express');
+
+//Can remove all session related items ahead of configing tokens
+
+// const session = require('express-session');
+
+// const KnexSessionStore = require('connect-session-knex')(session);
+
+const jwt = require('jsonwebtoken');
+
+const cors = require('cors');
+
+const db = require('./database/dbConfig.js');
+
+const bcrypt = require('bcryptjs');
+
+const server = express();
+
+const cookieTime = (req, res, next) => {
+    const token = req.headers.authorization;
+
+    if(token) {
+        jwt.verify(token, jwtSecret, (err, decodedToken) => {
+            if(err) {
+                res.status(401).json({ message: "Invalid toke, Take another"})
+            } else {
+                req.decodedToken = decodedToken;
+                next();
+            }
+        })
+    }
+    else {
+        res.status(401).json({message: 'Take a toke.'})
+    }
+
+    // next();
+//   if (req.session && req.session.username) {
+//     next();
+//     } else {
+//       res.status(401).send('Not authorized');
+//     }
+  }
+
+
+// const sessionConfig = {
+//     name:'notsession', ///we want to change to anon session software
+//     secret:'something%dwarf#something',
+//     cookie: { 
+//       secure: false,
+//       maxAge: 1000 * 60 * 1
+//     },
+//     httpOnly: true, ///block JS
+//     resave: false,
+//     saveUninitialized: false,
+//     store: new KnexSessionStore({
+//       tablename: 'sessions',
+//       sidefieldname: 'sid',
+//       knex: db,
+//       createtable: true,
+//       clearIntervale: 1000 * 60 * 60,
+//     }),
+// };
+
+
+const jwtSecret = 'quikbrownfox';
```

Comment:
Put secrets in environment variables so they aren't visible on github. Even though it's not as important on this project, it's a good thing to make a habit of.